Snakemake/workflow/rules/prepare_files_for_tsinfer.smk

```diff
@@ -64,21 +63,27 @@ rule extract_vcf_pos:
     # bcftools query -f '%CHROM %POS\n' {input} > {output.sites}
     #     awk '{{print $1"_"$2}}' {output.sites} > {output.file}
 
+if config['ancestralAllele'] not null:
```

Comment:
I think this one should not be neccessary? We have somewhere in the code that if it is null, then it runs the AA bit